cypress/utils/index.js

```diff
@@ -0,0 +1,79 @@
+import { waitForGQL } from "./networking";
+
+// used to test status and base status dropdown filters
+export const clickingCheckboxFetchesFilteredTasksAndUpdatesUrl = ({
+  selector = ".cy-checkbox",
+  checkboxDisplayName,
+  pathname,
+  paramName,
+  search,
+  query: { name, responseName, requestVariables },
+}) => {
+  cy.get(selector)
+    .contains(checkboxDisplayName)
+    .as("target")
+    .click();
+  resultsAreFechedAndRendered({
+    queryName: name,
+    responseName,
+    requestVariables,
+  }).then(() => urlSearchParamsAreUpdated({ pathname, paramName, search }));
+  cy.get("@target").click({ force: true });
+  resultsAreFechedAndRendered({
+    queryName: name,
+    responseName,
+    requestVariables: [],
+  }).then(() =>
+    urlSearchParamsAreUpdated({ pathname, paramName, search: null })
+  );
+};
+
+export const assertQueryVariables = (queryName, variables = {}) => {
+  const options = {};
+  Object.entries(variables).forEach(([variable, value]) => {
+    if (variable.includes("status")) {
```

Comment:
checking the variable name seems fragile since the variable name could change. we can change this condition to `Array.isArray(value)` and handle all arrays the same way.